RELEASE_NOTES.txt

```diff
@@ -24,6 +24,9 @@ Improvements
 ------------
  - setup.py detects conda env and uses installed conda (hdf5, bzip2, lzo
    and/or blosc) packages when building from source.
+ - On interactive python sessions, group/attribute  `__dir__()` method
```

Comment:
This was not released yet. `v3.4.2` does not include PR 624 and PR 625. Please move this to "Changes from 3.4.2 to 3.4.x".